ICSharpCode.CodeConverter/VB/NodesVisitor.cs

```diff
@@ -911,13 +911,13 @@ public override VisualBasicSyntaxNode VisitParenthesizedExpression(CSS.Parenthes
         public override VisualBasicSyntaxNode VisitPrefixUnaryExpression(CSS.PrefixUnaryExpressionSyntax node)
         {
             var kind = CS.CSharpExtensions.Kind(node).ConvertToken(TokenContext.Local);
-            if (IsReturnValueDiscarded(node)) {
-                return SyntaxFactory.AssignmentStatement(
-                    kind,
-                    (ExpressionSyntax)node.Operand.Accept(TriviaConvertingVisitor),
-                    SyntaxFactory.Token(VBUtil.GetExpressionOperatorTokenKind(kind)), _commonConversions.Literal(1)
-                );
-            }
+            //if (IsReturnValueDiscarded(node)) {
+            //    return SyntaxFactory.AssignmentStatement(
+            //        kind,
+            //        (ExpressionSyntax)node.Operand.Accept(TriviaConvertingVisitor),
+            //        SyntaxFactory.Token(VBUtil.GetExpressionOperatorTokenKind(kind)), _commonConversions.Literal(1)
+            //    );
+            //}
```

Comment:
If this bit needs to go, just delete it. We can always get it back from source control if we figure out what on earth it was trying to achieve at some point in the future (it's not obvious to me) 😄